packages/jest-matcher-utils/src/Replaceable.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import getType = require('jest-get-type');
+
+const supportTypes = ['map', 'array', 'object'];
+
+type ReplaceableForEachCallBack = (value: any, key: any, object: any) => void;
```

Comment:
can all these `any`s be `unknown`s?

Reply:
If change `any` to `unknown`, it will do a lot task for type converting.
For example:
```javascript
get(key: unknown): unknown {
  if (isMap(this.object)) {
    return this.object.get(key);
  } else if (Array.isArray(this.object)) {
    return this.object[key as number];
  } else if (typeof this.object === 'object' && this.object !== null) {
    return this.object[key as keyof object];
  }
}
```
I think it make code a little bit ugly, although is suitable for typescript.